9/problem9.rb

```diff
@@ -0,0 +1,66 @@
+require 'date'
+require 'time_difference'
+
+puts "Enter your Birth date:(DD/MM/YYYY)"
+
+b_date=gets.chomp!
+
+splitted_b_array = b_date.split("/")
+
+bday = Integer(splitted_b_array[0])
+
+bmonth = Integer(splitted_b_array[1])
+
+byear = Integer(splitted_b_array[2])
+
+
+currentDate=DateTime.now
+createdDate=Date.new(byear,bmonth,bday)
+
+diff = TimeDifference.between(createdDate, currentDate).humanize
+splitted_diff=diff.split(" ")
```

Comment:
`diff = TimeDifference.between(createdDate, currentDate).in_years`
if you just want number of years, you can use above line.